srcpkgs/papis/template

```diff
@@ -1,21 +1,25 @@
 # Template file for 'papis'
 pkgname=papis
-version=0.8
-revision=3
+version=0.9
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-requests python3-yaml python3-chardet python3-BeautifulSoup4
  python3-colorama python3-click python3-slugify python3-prompt_toolkit2
  python3-tqdm python3-Pygments python3-stevedore python3-parsing
  python3-filetype python3-bibtexparser python3-habanero python3-arxiv2bib
- python3-pylibgen python3-isbnlib python3 python3-setuptools"
+ python3-isbnlib python3 python3-setuptools python3-lxml
+ python3-typing_extensions python3-doi"
 short_desc="Command-line based document and bibliography manager"
 maintainer="xaltsc <xaltsc@protonmail.ch>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/papis/papis"
 distfiles="https://github.com/papis/papis/archive/v${version}.tar.gz"
-checksum=6c5493d5f7062a960bc2b1351ddde001ed585e407aa06e7c07092c61040c696a
+checksum=c49e53f843a53ccaaeaa24efa777c444efcfd541098fefcfa198ab99888b15e5
 
 pre_build() {
+        patch -p 1 -i "${FILESDIR}/update-vers1.patch"
```

Comment:
Patches put into `patches` directory are applied without need to modify template. They need to be in -p0 format, so please remove `a/` from header.